Fix #961: Remove root_dir args and visualization from evaluate_image_boxes and evaluate_boxes function

Samia35-2973 opened the pull request:
This addresses issue #961 by making the following changes:

- Removed the root_dir argument from both the evaluate_image_boxes and evaluate_boxes functions as it was unnecessary.
- Removed the visualization component in the evaluate_image_boxes function

henrykironde commented:
Under review:  [ff66fac](https://github.com/weecology/DeepForest/pull/964/commits/ff66fac9e277a38fdcca9e97fba7def479bae2c7)

henrykironde reviewed (changes requested):
Thanks @Samia35-2973  for getting to work on this issue. I expected some changes in the tests too.

Samia35-2973 commented:
Thank you for the feedback @henrykironde. I am working on the test cases.

Samia35-2973 commented:
- I’ve updated **evaluate.py** by removing the `root_dir` arguments, as I had previously missed a few instances. 
- Additionally, I modified **test_evaluate.py** as requested, removing `root_dir` from the calls to `evaluate_image_boxes()` and `evaluate_boxes()`.
- In the `test_evaluate_boxes_save_images` method, I also removed the assertion checking for saved images, since `evaluate_boxes()` no longer saves comparison files.

All tests pass locally after these updates. Please let me know if anything needs further clarification or if there are any additional changes you'd like me to make.

henrykironde commented on tests/test_evaluate.py:
```diff
@@ -72,9 +69,7 @@ def test_evaluate_boxes_save_images(tmpdir):
                                       ground_df=ground_truth,
-                                      root_dir=os.path.dirname(csv_file),
                                       savedir=tmpdir)
-    assert all([os.path.exists("{}/{}".format(tmpdir, x)) for x in ground_truth.image_path])
```
Why are we removing this assert?
